packages/helm-charts/oracle/CELOBRL.yaml

```
@@ -11,21 +11,41 @@ oracle:
   apiRequestTimeoutMs: 5000
   circuitBreakerPriceChangeThreshold: 0.25
   priceSources: "[
-      [
-        { exchange: 'BINANCE', symbol: 'CELOBUSD', toInvert: false },
-        { exchange: 'BINANCE', symbol: 'BUSDBRL', toInvert: false }
-      ],
-      [
-        { exchange: 'COINBASE', symbol: 'CELOUSD', toInvert: false},
-        { exchange: 'BITSO', symbol: 'USDBRL', toInvert: false }
-      ],
-      [
-        { exchange: 'COINBASE', symbol: 'CELOBTC', toInvert: false },
-        { exchange: 'NOVADAX', symbol: 'BTCBRL', toInvert: false }
-      ]
+    [
+      { exchange: 'BINANCE', symbol: 'CELOBUSD', toInvert: false },
+      { exchange: 'BINANCE', symbol: 'BUSDBRL', toInvert: false }
+    ],
+    [
+      { exchange: 'BINANCE', symbol: 'CELOUSDT', toInvert: false },
+      { exchange: 'BINANCE', symbol: 'USDTBRL', toInvert: false }
+    ],
+    [
+      { exchange: 'BINANCE', symbol: 'CELOBTC', toInvert: false },
+      { exchange: 'MERCADO', symbol: 'BTCBRL', toInvert: false }
+    ],
+    [
+      { exchange: 'BINANCEUS', symbol: 'CELOUSD', toInvert: false },
+      { exchange: 'BITSO', symbol: 'USDBRL', toInvert: false }
+    ],
+    [
+      { exchange: 'COINBASE', symbol: 'CELOUSD', toInvert: false},
+      { exchange: 'BITSO', symbol: 'USDBRL', toInvert: false }
+    ],
+    [
+      { exchange: 'COINBASE', symbol: 'CELOBTC', toInvert: false },
+      { exchange: 'NOVADAX', symbol: 'BTCBRL', toInvert: false }
+    ],
+    [
+      { exchange: 'OKX', symbol: 'CELOUSDT', toInvert: false },
+      { exchange: 'BINANCE', symbol: 'USDTBRL', toInvert: false } 
```

```suggestion
      { exchange: 'OKX', symbol: 'BTCUSDT', toInvert: ture },
       { exchange: 'NOVADAX', symbol: 'BTCBRL', toInvert: false }
```

I would suggest using this implicit route. It is high volume and works for US based oracle clients.